package exceptionHandling;

public class TryCatchBlock {

	String name;
	public static void main(String[] args) {
	int a=9;
	int b=0;
	
      System.out.println("hello");
      try
      {
    	  TryCatchBlock bt =new TryCatchBlock();
    	  bt =null;
    	 bt.name ="Vijay";
    	  //int c= a/b;
    	  System.out.println("111111");
    	  
      }
      catch (ArithmeticException e)
      {
    	  System.out.println("Hi AE is comming...");
    	  e.printStackTrace();
    	
      }catch(NullPointerException e)
      {
    	  System.out.println("Hi NE is comming...");
    	  e.printStackTrace();
      }
     
	}

}
